src/applications/vaos/services/appointment/transformers.js

```diff
@@ -402,7 +403,13 @@ function setContained(appt) {
           ],
         };
 
-        if (appt.sta6aid) {
+        if (tasInfo) {
+          service.location = tasInfo.address;
```

Comment:
Even though we only have the address, I think we're want this `location` field to be an object that follows the [FHIR Location schema](https://www.hl7.org/fhir/dstu2/location.html).  Tthe address within also needs to follow the [Address datatype](https://www.hl7.org/fhir/dstu2/datatypes.html#Address), so you might want to do something like this:

```js
const { address } = tasInfo;
service.location = {
  address: {
    line: [address.streetAddress],
    city: address.city,
    postalCode: address.postalCode,
    country: address.country
  },
};
```

You can see an example of a location mapping in `src/applications/vaos/services/location/transformers.js`